pep-0690.rst

```diff
@@ -426,9 +436,10 @@ rules.
 
 The more difficult case can occur if an import in third-party code that can't
 easily be modified must be forced to be eager. For this purpose,
-``importlib.set_lazy_imports()`` takes an optional keyword-only ``excluding``
-argument, which can be set to a container of module names within which all
-imports will be eager::
+``importlib.set_lazy_imports()`` takes two optional arguments: a boolean, ``True``
```

Comment:
Should both options be keyword-only?

I ask because in general, boolean non-keyword arguments are often pretty hard to understand.  What does the boolean refer to, and how can you tell without looking the function up?  I personally find position booleans to be poor API design.

However, in *this* case, maybe it's okay.  What you'd be writing is `set_lazy_imports(False)` and that *is* rather self explanatory.  So maybe it's okay?

Reply:
Yes, we had a discussion about this, internally, and we thought `set_lazy_imports(False)` and `set_lazy_imports(True)` to be so simple and self explanatory enough that people could actually use it without confusion, yet they can also be explicit and use `set_lazy_imports(enable=True)`.